packages/database/src/migrations/20210520020331-AddAccessTokenExpiryFieldToTupaiaUserSessionTable-modifies-schema.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+    ALTER TABLE "userSession" 
+    ADD COLUMN access_token_expiry BIGINT NOT NULL DEFAULT 0;
```

Comment:
Bit of an odd decision for me to choose the default of 0 here. Basically, didn't wanna just make it `NOT NULL` as then I'd have to track down and refactor everywhere that these sessions get created. This way, sessions can be created without an expiry and then whenever a piece of code that uses the expiry logic gets called it immediately refreshes and sets the expiry for the first time.

Reply:
Nice :+1: